### PR TITLE
Extract minimum required coverage by Jacoco plugin.

### PR DESCRIPTION
**Pull request by maciejwalkowiak**
## :loudspeaker: Type of change

- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement
- [x] Refactoring

## :scroll: Description

Move minimum required coverage to `Config` class and use same value (`0.6`) in all modules.

## :bulb: Motivation and Context

Reduce duplication and possibility of having different quality gates in different modules.

## :pencil: Checklist

- [x] I reviewed submitted code
- [ ] I added tests to verify changes
- [x] All tests passing
- [x] No breaking changes

### CODE REVIEW

**Review by marandaneto — APPROVED** [review]
LGTM

File: buildSrc/src/main/java/Config.kt [hunk 1]
```diff
@@ -72,6 +74,10 @@ object Config {
     }
 
     object QualityPlugins {
+        object Jacoco {
+            val version = "0.8.5"
+            val minimumCoverage = BigDecimal.valueOf(0.6)
+        }
         val jacocoVersion = "0.8.5"
```
Comment (bruno-garcia) [inline buildSrc/src/main/java/Config.kt:81]: This should be deleted?

**Comment by bruno-garcia** [comment]
Thanks for this

File: buildSrc/src/main/java/Config.kt (hunk 1 above)
Comment (maciejwalkowiak) [reply buildSrc/src/main/java/Config.kt:81]: Good catch. I'll remove it in Spring PR.

**Pull request merged**